wordpress-mobile/WordPress-iOS
# Stats: add File Downloads to Core Data

Fixes #11963 
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/175

To test:
- Go to Stats period.
  - Verify File Downloads now has data. (This verifies the WPKit change.)
  - FD is not live in Calypso yet. Data can be verified via the console with the `file-downloads` endpoint. Ex: `/sites/<site_id>/stats/file-downloads?period=day&date=2019-08-05&num=1`
  - The filenames are displayed as-is. I've inquired on the FD P2 exactly how filenames are going to be displayed/formatted in Calypso, and will make any adjustments with a future change if necessary.

<img width="400" alt="file_downloads" src="https://user-images.githubusercontent.com/1816888/62502502-030e3a00-b7ad-11e9-9c15-da5b81c85c09.png">

- Disable your network.
  - Return to stats.
  - Verify File Downloads has data.

Update release notes:
